build/scripts/test-determinism.ps1

```diff
@@ -1,6 +1,7 @@
 [CmdletBinding(PositionalBinding=$false)]
 param ( [string]$bootstrapDir = "",
-        [switch]$debugDeterminism = $false)
+        [switch]$release = $false,
+        [string]$altRootDrive = "q:")
```

Comment:
Please document what "altRootDrive" is.

Reply:
(there's a good comment at the use site 200 lines away...might be good to put it here at the top too.)